packages/gatsby-plugin-netlify-cms/package.json

```diff
@@ -35,7 +35,9 @@
   "main": "index.js",
   "peerDependencies": {
     "gatsby": "^2.0.101",
-    "netlify-cms": "^2.0.6"
+    "netlify-cms-app": "^2.9.0",
+    "react": "^16.8.4",
```

Comment:
NetlifyCMS has a peer dependency on `^16.8.x`, for version 2.9.x on this new lib (netlify-cms-app) , so I wanted to keep that since there was a breaking change anyway. If we drop the version we will get warnings in this plugin and we may cause invariant errors when/if using hooks in the cms widget plugins. Better to leave this in.